src/actions/breakpoints.js

```diff
@@ -12,17 +12,14 @@ const constants = require("../constants");
 const { PROMISE } = require("../utils/redux/middleware/promise");
 const { getBreakpoint, getBreakpoints, getSource } = require("../selectors");
 
-const {
-  getOriginalLocation, getGeneratedLocation, isOriginalId
-} = require("../utils/source-map");
+const { getOriginalLocation, getGeneratedLocation, isOriginalId } = require(
+  "../utils/source-map"
+);
 
-import type { ThunkArgs } from "./types";
-import type { Location } from "../types";
+import type {ThunkArgs} from "./types";
+import type {Location} from "../types";
```

Comment:
That is fixed on master. Not sure if it was included in the latest published version or not.